docs: Remove deprecated eslintrc documentation files (#20472)
eslint/eslint: docs/src/use/configure/migration-guide.md

};
```

> You would do the same thing like this:

Reviewer: ```suggestion
In flat config, you would do the same thing like this:
```

};
```

> You would do the same thing like this:

Reviewer: ```suggestion
In flat config, you would do the same thing like this:
```

### Processors

> Processors can be referenced from plugins by their name, or they can be inserted directly into the configuration. Processors must be explicitly set in the configuration.

Reviewer: ```suggestion
In eslintrc files, processors had to be defined in a plugin, and then referenced by name in the configuration. Processors beginning with a dot indicated a file extension-named processor which ESLint would automatically configure for that file extension.

In flat config files, processors can still be referenced from plugins by their name, but they can now also be inserted directly into the configuration. Processors will _never_ be automatically configured, and must be explicitly set in the configuration.
```
